sklearn/feature_selection/tests/test_from_model.py

```diff
@@ -101,6 +103,48 @@ def test_feature_importances():
     mask = np.abs(transformer.estimator_.coef_) > 1e-5
     assert_array_equal(X_new, X[:, mask])
 
+    # Check with various values of k
```

Comment:
I would suggest putting these tests in a separate function, because they're testing a different functionality of the code.